bin/e2e-test-client/src/test_context.rs

```diff
@@ -176,7 +176,11 @@ impl Wallet {
             amount: 0,
             asset_id,
         });
-        tx.with_params(self.consensus_params);
+        tx.with_tx_params(self.consensus_params.tx_params);
```

Comment:
We need to support continue supporting `tx.with_params(self.consensus_params);`=) Otherwise it is a lot of boilerplate code